### PR TITLE
hotfix clippy 1.72 lints

### PR DESCRIPTION
**Pull request by dyc3**
This fixes the recent CI failures we've been getting.

### CODE REVIEW

**Comment by AviiNL** [comment]
Clippy errors on it for a reason, no? This feels more like a duct tape patch than an actual fix, I'm fine with getting this merged in as a hotfix, but shouldnt the underlying problem not get fixed instead?

File: crates/valence_text/src/tests.rs [hunk 1]
```diff
@@ -64,7 +64,7 @@ fn selector() {
     let deserialized = Text::from_str(&serialized).unwrap();
     assert_eq!(
         serialized,
-        r##"{"selector":"foo","separator":{"text":"bar","color":"red","bold":true}}"##
```
Comment (AviiNL) [inline crates/valence_text/src/tests.rs:67]: this looks unrelated, should this be in this pr?

Comment (dyc3) [reply crates/valence_text/src/tests.rs:67]: This was caught in a clippy lint when I ran it.

Comment (AviiNL) [reply crates/valence_text/src/tests.rs:67]: okay 👍 
LGTM then

**Comment by rj00a** [comment]
> Clippy errors on it for a reason, no? This feels more like a duct tape patch than an actual fix, I'm fine with getting this merged in as a hotfix, but shouldnt the underlying problem not get fixed instead?

Snbt module is in need of a full rewrite so I'm fine with this temp fix.
